### PR TITLE
Feature/spm

### PR DESCRIPTION
**Pull request by anhar**
Hi @vexy 👋 

I know this is only your fork from dragstor but I saw that he has not merged your pull request so I thought I should try and get this merged here instead.

**Changelog:**

- Removed Carthage
- Added Swift Package manager as the default dependency manager for handling dependencies
- Fixed a compilation issue in TimerController due to update of SQLite library

### CODE REVIEW

**Comment by vexy** [comment]
Hey @anhar, thank you for this contribution ! ❤️ 
I appreciate taking time dealing with this. I'll go ahead, review this PR and merge it eventually.

Believe it or not, I personally know @dragstor, and he is just a busy dad nowadays 😄 .
We don't mind him not reviewing our PRs 😈

Anyhow, if you're willing to contribute more, just let me know. I'd be happy to hear new things and (re)kick-off `freelancers dashboard` once again. The days of `SwiftUI` and `Big Sur` arrived. We could give it a shot here if you guys will. @dragstor @anhar.

File: Freelancer/Controller/Timer/TimerController.swift [hunk 1]
```diff
@@ -84,10 +84,18 @@ class TimerController: NSViewController {
                         ts_approved <- false
                     )
                 )       //extract method
-            } catch let Result.error(message, code, statement) where code == SQLITE_ANY {
```
Comment (vexy) [inline Freelancer/Controller/Timer/TimerController.swift:87]: Nice things removing redundant `where` check. 👍

**Review by vexy — APPROVED** [review]
Nice cleanup and refreshment PR. Good job overall 👍

**Comment by vexy** [comment]
@anhar you can go ahead and (squash) merge this PR. I prefer squashing over rebasing, if you feel otherwise just do it :)

**Comment by dragstor** [comment]
Hey guys :smile: 

Apologies to @vexy for not merging. It somehow slipped away. Will do it now.

At this very moment, I'm into another huge project of mine + baby + day job, so I kind of put all other side projects on hold :hourglass: Merge all of those together, you get this haha :rofl: 

I'm happy there's some interest in the Freelancer's Dashboard!

@vexy @anhar  Guess you can finish with this merge, and perhaps we could do a PR on the original repo as well :heart: 

Cheers!

**Comment by anhar** [comment]
@vexy I squashed the commits as you suggested. It's already based on the latest master branch so it won't require any git rebase. You'll have to approve and merge it as it is your repository 😄 

And @dragstor congrats on the baby! 🎆

**Comment by vexy** [comment]
Done !

**Pull request merged**